src/Collection.js

```diff
@@ -200,16 +200,13 @@ export default class Collection<T: Model> {
 
     try {
       data = await promise
-    } catch (body) {
+    } catch (err) {
       runInAction('create-error', () => {
-        if (model) {
-          this.remove([model.id])
-        }
-        this.error = { label, body }
+        if (model) this.remove([model.id])
+
+        this.error = { label, body: err }
         this.request = null
       })
-
-      throw body
```

Comment:
Any ajax errors? I mean neither 5xx ? This is an open discussion we had yesterday